Automattic/wp-calypso
# Account Recovery: Add the reset password email form page

## Summary
![image](https://cloud.githubusercontent.com/assets/1842898/23497154/c1071baa-ff5c-11e6-9e78-b5c7873c8b8d.png)
This PR adds the page a user would see once the password reset link has been sent successfully. Note that the `email` is temporarily commented and replaced by a placeholder just for now.

## Test Plan
* Open an incognito window and make sure you are logged out.
* Visit http://calypso.localhost:3000/account-recovery/reset-password/email-form and make sure the above page is properly shown.
* The **You didn't receive it** link won't do anything for now.
